variables.tf

```diff
@@ -228,7 +229,6 @@ variable "private_endpoints" {
     }), null)
     tags                                    = optional(map(string), null)
     subnet_resource_id                      = string
-    subresource_name                        = string
```

Comment:
Not sure we are allowed to chane the format of the private endpoints variable thi is constant among all the avm resources.

Reply:
since it'll always be amlworkspace for us, I think it makes sense to remove from vars